### PR TITLE
Remove jsx attribute from item scope

### LINKED ISSUE #2034: comma based item broken inside of html attributes
Can't target `"disabled": disabled` or `"invalid": invalid` since item now refers to an html attribute

```jsx
<div className={classNames({"disabled": disabled, "invalid": invalid})} />
```

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
Fixes #2034

## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [-] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

**Review by pokey — COMMENTED** [review]
We should prob ask on slack whether anyone is using this. Or maybe you already did that?

**Comment by AndreasArvidsson** [comment]
> We should prob ask on slack whether anyone is using this. Or maybe you already did that?
No I haven't

File: packages/cursorless-engine/src/languages/html.ts
Comment (pokey) [inline packages/cursorless-engine/src/languages/html.ts:1]: 🎉

**Review by pokey — APPROVED** [review]
Looks good but please file an issue to track re-adding if there's not already one

**Comment by AndreasArvidsson** [comment]
I'm not completely sure if we actually want that? it's one of those things that if we overload a single scope to much it gets imprecise. I can make an issue about thinking about it :D

**Pull request merged**